Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.cpp

```diff
@@ -338,16 +382,24 @@ MacroscopicProperties::InitData ()
         Bz_IndexType[idim]      = Bz_stag[idim];
         macro_cr_ratio[idim]    = 1;
 #ifdef WARPX_MAG_LLG
-        Hx_IndexType[idim]             = Hx_stag[idim];
-        Hy_IndexType[idim]             = Hy_stag[idim];
-        Hz_IndexType[idim]             = Hz_stag[idim];
-        mag_alpha_IndexType[idim]      = mag_alpha_stag[idim];
-        mag_gamma_IndexType[idim]      = mag_gamma_stag[idim];
-        Mx_IndexType[idim]             = Mx_stag[idim];
-        My_IndexType[idim]             = My_stag[idim];
-        Mz_IndexType[idim]             = Mz_stag[idim];
-        mag_exchange_IndexType[idim]   = mag_exchange_stag[idim];
-        mag_anisotropy_IndexType[idim] = mag_anisotropy_stag[idim];
+        Hx_IndexType[idim]              = Hx_stag[idim];
+        Hy_IndexType[idim]              = Hy_stag[idim];
+        Hz_IndexType[idim]              = Hz_stag[idim];
+        Mx_IndexType[idim]              = Mx_stag[idim];
+        My_IndexType[idim]              = My_stag[idim];
+        Mz_IndexType[idim]              = Mz_stag[idim];
+        mag_alphax_IndexType[idim]      = mag_alphax_stag[idim];
```

Comment:
Is mag_alphax_IndextType used anywhere?
and same with the gammax, exchanges and anisotropy

Reply:
none of these are used at present; would you prefer we delete them until they are needed?

Reply:
removed in 4ce3a7c70ee4a5a674c69d883fdfa32f7f84b5c7